ql/src/java/org/apache/hadoop/hive/ql/exec/tez/WorkloadManager.java

```diff
@@ -2225,13 +2251,17 @@ private void resetAndQueueKill(Map<WmTezSession, KillQueryContext> toKillQuery,
     KillQueryContext killQueryContext, Map<WmTezSession, GetRequest> toReuse) {
 
     WmTezSession toKill = killQueryContext.session;
+    String poolName = toKill.getPoolName();
+    boolean validPoolName = poolName != null;
```

Comment:
break this line into 2. so that code will be more readable.

Reply:
Broke the code into two separate blocks